server/cluster.go

```diff
@@ -148,6 +148,10 @@ func encodeRegionSearchKey(endKey []byte) string {
 	return string(append([]byte{'z'}, endKey...))
 }
 
+func encodeRegionStartKey(startKey []byte) string {
```

Comment:
@qiuyesuifeng No. Separate them guarantees that 'encoded startKey nil' < 'encoded endKey nil'. StartKey and EndKey will be compared at L262.